README.md

```diff
@@ -1,7 +1,10 @@
 
 
 # customs-update-eori-admin-acceptance-tests
-UI test suite for the `<digital service name>` using WebDriver and `<scalatest/cucumber>`.  
+UI test suite for the `EORI Toolkit` using WebDriver and `<scalatest/cucumber>`.  
```

Comment:
I do not think this repo uses cucumber, it only uses scalatest and Webdriver. the When and Then syntax is using the scalatest library.